content/en/platform/corda/4.10/enterprise/tools-config-obfuscator.md

```diff
@@ -21,6 +21,10 @@ The Configuration Obfuscator makes node installation less vulnerable to someone
 Although the Configuration Obfuscator does protect the node against trawling attacks, it does not ensure that password protection is completely secure. For improved security, always use the latest released version of the tool.
 {{< /warning >}}
 
+{{< note >}}
+Obfuscation is supported exclusively for Corda-related configuration files, such as node or other core components. It is not available for CorDapps or non-Corda-specific configuration files.
+{{< /note >}}
+
 {{< note >}}
 The Configuration Obfuscator tool can only be used with configuration files for Corda Enterprise Edition 4.4 (and above) nodes, Corda Firewall, and Corda Network Enterprise Manager (CENM) 1.3 (and above) services. Corda configuration files obfuscated with older versions of Corda Enterprise can still be deobfuscated by Corda Enterprise Edition 4.4 and above. Configuration files obfuscated with CENM 1.1 and 1.2 can be deobfuscated with CENM 1.3 (and above), but further obfuscation must be done using the new version of the Configuration Obfuscator tool.
```

Comment:
Can we just add one sentence to this note saying "Obfuscation is not supported in CorDapp configuration files." ?

Reply:
Added.